### PR TITLE
Release 0.6.0

### PR DESCRIPTION
**Pull request by diegofiori**

### CODE REVIEW

File: nebullvm/operations/measures/utils.py [hunk 1]
```diff
@@ -4,13 +4,13 @@
 
 import numpy as np
 
-from nebullvm.base import Device
+from nebullvm.tools.base import Device
```
Comment (valeriosofi) [inline nebullvm/operations/measures/utils.py:7]: sort imports

File: nebullvm/operations/optimizations/compilers/openvino.py [hunk 2]
```diff
@@ -0,0 +1,150 @@
+from pathlib import Path
+import subprocess
```
Comment (valeriosofi) [inline nebullvm/operations/optimizations/compilers/openvino.py:2]: sort imports

File: nebullvm/operations/optimizations/compilers/tensor_rt.py [hunk 3]
```diff
@@ -0,0 +1,365 @@
+import abc
+import os
+from pathlib import Path
+import subprocess
```
Comment (valeriosofi) [inline nebullvm/operations/optimizations/compilers/tensor_rt.py:4]: sort imports

File: nebullvm/tools/pytorch.py [hunk 4]
```diff
@@ -1,9 +1,9 @@
 from pathlib import Path
 from typing import List, Tuple, Optional, Dict, Union, Sequence
 
-from nebullvm.base import DataType, InputInfo, Device
+from nebullvm.tools.base import DataType, InputInfo, Device
```
Comment (valeriosofi) [inline nebullvm/tools/pytorch.py:4]: sort imports

File: nebullvm/tools/tf.py [hunk 5]
```diff
@@ -1,6 +1,6 @@
 from typing import Union, List, Tuple, Any, Optional, Dict
 
-from nebullvm.base import InputInfo, DataType, Device
+from nebullvm.tools.base import InputInfo, DataType, Device
```
Comment (valeriosofi) [inline nebullvm/tools/tf.py:3]: sort imports

**Review by valeriosofi — CHANGES_REQUESTED** [review]

File: apps/accelerate/speedster/speedster/api/tests/test_tensorflow.py [hunk 6]
```diff
@@ -3,27 +3,32 @@
...
-from nebullvm.inference_learners.onnx import TensorflowONNXInferenceLearner
-from nebullvm.inference_learners.openvino import (
+from speedster import optimize_model
+
+
+# Limit tensorflow gpu memory usage
+from nebullvm.operations.inference_learners.onnx import (
+    TensorflowONNXInferenceLearner,
+)
+from nebullvm.operations.inference_learners.openvino import (
```
Comment (diegofiori) [inline apps/accelerate/speedster/speedster/api/tests/test_tensorflow.py:16]: wrong import order

**Pull request merged**